src/components/KMultiselect/KMultiselect.vue

```diff
@@ -0,0 +1,911 @@
+      hiddenItemsTooltip,
+      // style and attributes
+      widthStyle,
+      inputWidth,
+      modifiedAttrs,
+      isHovered,
+      isFocused,
+      isDisabled,
+      isReadonly,
+      popper,
+      boundKPopAttributes,
+      // functions
+      sortItems,
+      handleItemSelect,
+      clearSelection,
+      triggerFocus,
+      onQueryChange,
+      onInputFocus,
+      onPopoverOpen,
+    }
+  },
+})
+</script>
+
+<style lang="scss" scoped>
+@import '@/styles/variables';
+@import '@/styles/functions';
+
+.k-multiselect {
+  width: fit-content; // necessary for correct placement of popup
+
+  // off screen area for checking selections before display
+  .staging-area {
+    position: absolute;
+    left: -99999px;
+  }
+
+  .k-multiselect-selections {
+    --KBadgeMaxWidth: 100px;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    padding-left: 16px;
+    padding-right: 23px;
+
+    &.staging {
+      -webkit-box-sizing: border-box;
+      -moz-box-sizing: border-box;
+      box-sizing: border-box;
+      position: relative;
+      height: auto;
+      padding-left: 16px;
+      padding-right: 23px;
+    }
+
+    .hidden-selection-count {
+      // match dismissable height
+      --KBadgeLineHeight: 21px;
+
+      &.hidden {
+        visibility: hidden;
+      }
+    }
+  }
+
+  .k-multiselect-icon {
+    position: absolute;
+    z-index: 1;
+    right: 1px;
+    top: 1px;
+
+    .k-multiselect-chevron-icon {
+      position: relative;
+      top: 11px;
+      right: 10px;
+    }
+
+    .k-multiselect-clear-icon {
+      position: relative;
+      top: 8px;
+      right: 10px;
+    }
+  }
+
+  .k-multiselect-trigger {
+    display: inline-block;
+    position: relative;
+    // mimic input's box shadow styling
+    background-color: var(--KInputBackground, var(--white));
+    box-shadow: inset 0 0 0 1px var(--KInputBorder, var(--grey-300)) !important;
+    border-radius: 3px;
+
+    &.hovered {
+      box-shadow: inset 0 0 0 1px var(--KInputHover, var(--blue-200)) !important;
+    }
+
+    &.focused {
+      box-shadow: inset 0 0 0 1px var(--KInputFocus, var(--blue-400)) !important;
+    }
+
+    &.readonly,
+    &.disabled {
+      background-color: var(--KInputReadonlyBackground, var(--grey-100, #f8f8fa));
+      box-shadow: inset 0 0 0 1px var(--KInputBorder, var(--grey-300)) !important;
+    }
+
+    .k-multiselect-input {
+      position: relative;
+      display: inline-block;
+      width: 100%;
+    }
+  }
+}
+</style>
+
+<style lang="scss">
+@import '@/styles/variables';
+@import '@/styles/functions';
+
+.k-multiselect {
+  .k-multiselect-trigger {
+    .k-multiselect-input {
+      &.prevent-pointer-events {
+        pointer-events: none;
+      }
+
+      &.input-placeholder-dark::placeholder {
+        color: var(--KInputColor, var(--black-70, rgba(0, 0, 0, 0.7))) !important;
+      }
+
+      input.k-input:not([type=checkbox]):not([type=radio]) {
```

Comment:
question: since `KMultiselect` is always text, could you use that instead of the `:not` rules here?

Also, if you keep this, you should use the quotes in the selectors, e.g.
```
[type="checkbox"]
[type="radio"]
```